src/status_im/contexts/chat/messenger/composer/mentions/view.cljs

```diff
@@ -17,7 +17,7 @@
    user])
 
 (defn view
-  [{:keys [layout-height]}]
+  [layout-height]
   (let [suggestions                               (rf/sub [:chat/mention-suggestions])
```

Comment:
We passed a big map of props but we only used one of them, which is actually an `atom`, so it's a stable reference, no fear from uinexpected re-renders now. This is a simplification made.